app/controllers/order_details_controller.rb

```diff
@@ -72,6 +72,27 @@ def dispute
 
   private
 
+  def cancellation_success_path(order_detail)
+    if operator_cancellation?(order_detail)
+      facility_user_reservations_path(order_detail.facility, order_detail.user)
+    else
+      reservations_path
+    end
+  end
+
+  def cancel_with_fee(order_detail)
+    if operator_cancellation?(order_detail)
+      # When this is an operator who is not the ordering user; admin cancel with fee:
+      order_detail.cancel_reservation(session_user, OrderStatus.canceled_status, true, true)
```

Comment:
Agreed it's not a good interface. I didn't want to change it since the method is used elsewhere, but now that I'm looking for usage the cleanup would be less than I thought. I'll update.